config.yaml

```diff
@@ -4,12 +4,23 @@
 # openstackrc > token > password
```

Comment:
Should it rather use new configuration with config.schema.yaml? Having config.yaml is a trouble as st2 will read from it. @Kami may have example of multi-level config schema somewhere.